frontend/public/components/namespace.jsx

```diff
@@ -988,14 +989,17 @@ class NamespaceBarDropdowns_ extends React.Component {
       useProjects,
       children,
       disabled,
+      t,
     } = this.props;
     if (flagPending(canListNS)) {
       return null;
     }
 
     const { loaded, data } = this.props.namespace;
     const model = getModel(useProjects);
-    const allNamespacesTitle = `all ${model.labelPlural.toLowerCase()}`;
+    const allNamespacesTitle = t('dropdown~all {{models}}', {
+      models: model.labelPlural.toLowerCase(),
```

Comment:
If we're translating projects and namespaces in the nav, we probably want to translate it here, too. We should know if it's one or the other and use different messages since those are the only two possibilities.